package/files/nginx/nginx-http.conf

```diff
@@ -25,6 +25,10 @@ server {
         # return 301 https://$host$request_uri;
     }
 
+    location /public/tools {
```

Comment:
I wonder why we need show the  `index` on these pages. Do you explicitly navigate to them?

Reply:
No, customers download two files `rmt-client-setup` and `rmt-client-setup-res`. These would fail if the server service was failing.